tests/testthat/test-clone.R

```diff
@@ -1077,6 +1077,28 @@ test_that("Cloning inherited methods for non-portable classes", {
   expect_identical(a$x, 3)
 })
 
+test_that("Unknown environments are not copied nor $-inspected", {
+
+  `$.asdfasdf` <- function(x, value) {
+    stop("error")
+  }
+
+  bar <- R6Class("bar",
+                 public = list(
+                   x = NULL,
+                   initialize = function() {
+                     x <- new.env(parent = emptyenv())
+                     class(x) <- "asdfasdf"
+                     self$x <- x
+                     x
+                   }
+                 )
+  )
```

Comment:
Please indent like the rest of the code in these tests.

Reply:
Done